modules/edu.gemini.seqexec.server/build.sbt

```diff
@@ -0,0 +1,20 @@
+import Settings._
+import Settings.Libraries._
+import Settings.LibrariesJVM._
+
+name := "edu.gemini.seqexec.server"
+
+libraryDependencies ++= Seq(
+  ScalaZCore,
+  ScalaZConcurrent,
+  "io.argonaut"        %% "argonaut"                  % "6.1",
+  "commons-httpclient" % "commons-httpclient"         % "2.0",
+  // OCS bundles
+  SpModelCore,
+  POT,
+  EpicsACM
+)
+
+unmanagedJars in Compile ++= Seq(
+  new File((baseDirectory in ThisBuild).value, "lib/squants_2.11-0.6.1.jar")
```

Comment:
It is available as 0.5.3 but not 0.6.1. We are building the new version, so we have the binary. There is no ETA for 0.6.1 reaching maven central
I don't have a very strong opinion, but if we go with artifactory it worries me that it will try to reload it very often if it is a SNAPSHOT